products/stream/src/content/stream-connect/getting-started.md

```diff
@@ -26,21 +26,14 @@ You can do steps 1 and 2 using the API or the [Stream Connect UI](https://dash.c
 
 ### Create a live input
 
-This call allocates configuration for your RTMP(S) feed to be terminated at.
-
-The response from this endpoint contains the uid to identify the source, this is referred to as `$INPUT_UID` later on. It also contains the rtmps information required for broadcasting to this input.
-
-You may include additional metadata, such as the input’s name, they may do so by including a body on this request:
-`--data '{"meta": {"name": "test stream 1"}}'`
-
-Note that this metadata has no impact on Stream Connect's behavior, it is entirely for your bookkeeping.
-
+Create a live input to get an RTMPS URL to transmit live video to Cloudflare.
 
 #### cURL example
 ```bash
 curl -X POST \
 -H "Authorization: Bearer $TOKEN" \
-https://api.cloudflare.com/client/v4/accounts/$ACCOUNT/stream/live_inputs
+https://api.cloudflare.com/client/v4/accounts/$ACCOUNT/stream/live_inputs \
+--data '{"meta": {"name":"test stream 1"}}'
```

Comment:
Might be worth including at least a mention about the meta field, otherwise they may not realize it can be used for whatever they want.